package.json

```diff
@@ -74,7 +74,185 @@
           "when": "resourceLangId == 'feature'"
         }
       ]
-    }
+    },
+    "configuration": [
```

Comment:
suggestion: what about having version in the configuration?

Reply:
I don't know, what about it?

Reply:
I mean, it's a good idea, created an issue!

https://github.com/hedhyw/vscode-golang-gherkingen/issues/8